libbeat/template/load.go

```diff
@@ -148,7 +189,11 @@ func (l *Loader) LoadTemplate(templateName string, template map[string]interface
 
 // CheckTemplate checks if a given template already exist. It returns true if
 // and only if Elasticsearch returns with HTTP status code 200.
-func (l *Loader) CheckTemplate(templateName string) bool {
+// If no ES client is configured it returns false.
+func (l *Loader) templateLoaded(templateName string) bool {
+	if l.client == nil {
+		return false
+	}
 	status, _, _ := l.client.Request("HEAD", "/_template/"+templateName, "", nil, nil)
```

Comment:
Any reason to have changed the visibility of this method?
I understand the name could be better.

Reply:
It is not used anywhere else, so I there is no reason for exporting it.